components/automate-backend-elasticsearch/habitat/plan.sh

```diff
@@ -0,0 +1,48 @@
+# shellcheck disable=SC2148
+UPSTREAM_PKG_IDENT="chef/elasticsearch-odfe/0.10.1.2"
+pkg_name="automate-backend-elasticsearch"
+pkg_description="Wrapper package for elasticsearch-odfe"
+pkg_origin="chef"
+vendor_origin="chef"
+pkg_version="1.0.28"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
+pkg_license=("Chef-MLSA")
```

Comment:
Shouldn't this match the license of the upstream package and not our license?

Reply:
So even our upstream package is under Chef-MLSA license https://github.com/chef/opendistro-for-elasticsearch/blob/master/elasticsearch-odfe/plan.sh#L14

Reply:
That seems wrong from a licensing standpoint. We can't just relicense OSS software as we package it.

Reply:
@tas50 Changed upstream license https://github.com/chef/opendistro-for-elasticsearch/pull/15